pulsar-proxy/src/test/java/org/apache/pulsar/proxy/server/ProxyTest.java

```diff
@@ -222,7 +244,7 @@ public void testRegexSubscription() throws Exception {
             final int numMessages = 20;
 
             try (Producer<byte[]> producer = client.newProducer(Schema.BYTES)
-                .topic("persistent://sample/test/local/topic1")
+                .topic("persistent://sample/test/local/regex-sub-topic1")
```

Comment:
@rudy2steiner why this?

Reply:
I have a unit test case for this change and run the unit test ProxyTest. and found out the consumer of testRegexSubscription case will blocking because of consuming message from inconsistent topic with producer. maybe i should fix the issue in another pr.

Reply:
this issue has been fixed on master